pkg/skaffold/cluster/minikube.go

```diff
@@ -93,9 +103,46 @@ func minikubeExec(arg ...string) (*exec.Cmd, error) {
 	if err != nil {
 		return nil, fmt.Errorf("getting minikube executable: %w", err)
 	}
+
+	minVer, err := supportsUserFlagFunc()
+	if err != nil {
+		return nil, err
```

Comment:
Instead of returning err here, should we fall back to not adding minikube user flag.